app-dicts/aspell-es/aspell-es-1.11.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+ASPELL_LANG="Spanish"
+ASPELL_VERSION=6
+
+inherit aspell-dict-r1
+
+MY_P="${P/aspell/aspell6}-2"
```

Comment:
This substitution is needlessly complex. Explicit `aspell6-${PV}` would be shorter. Also please integrate that `-2` into the package version. You could e.g. make it `1.11.2` and use `ver_rs` (`eapi7-ver` eclass) to replace `.` with `-`, or e.g. do `1.11_p2`, then `${PV/_p/-}`.